nasl/nasl_frame_forgery.c

```diff
@@ -377,22 +377,22 @@ nasl_dump_frame (lex_ctxt *lexic)
  * @brief Get the MAC address of host
  *
  * @param[in] ip_address    Local IP address
+ * @param[out] mac    The mac address
  *
  * @return The MAC address of the host. NULL otherwise
```

Comment:
Needs to be updated to something like
```suggestion
 * @return 0 on success. MAC address is put into buffer. -1 on error.
```